bundles/com.espressif.idf.launch.serial.ui/src/com/espressif/idf/launch/serial/ui/internal/CMakeMainTab2.java

```diff
@@ -0,0 +1,282 @@
+/*******************************************************************************
+ * Copyright 2020 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
```

Comment:
As this is a copied class, please keep the original license and add espressif as a modifier

Reply:
@kolipakakondal changed